src/components/fx/click.js

```diff
@@ -21,7 +21,11 @@ module.exports = function click(gd, evt, subplot) {
     if(evt && evt.target) {
         if(gd._hoverdata) {
             data = gd._hoverdata;
-        } else if(clickmode.indexOf('anywhere') > -1) {
+        } else if(gd._fullLayout.geo){
```

Comment:
Shouldn’t this be inside the ˋ else if(clickmode.indexOf('anywhere') > -1) ˋ block?

Reply:
Otherwise it doesn’t matter that you set it to anywhere

Reply:
Yes, I'm so careless...